app/Providers/RouteServiceProvider.php

```diff
@@ -46,6 +46,10 @@ public function boot()
             Route::middleware('web')
                 ->namespace($this->namespace)
                 ->group(base_path('routes/web.php'));
+
+            Route::prefix('status')
+                ->namespace($this->namespace)
+                ->group(base_path('routes/status.php'));
```

Comment:
I'm not sure it's necessary to put this here. You can add prefixes directly in the routes file. Also by having these separate from web I'm not sure if they'll run through the wed middleware. Which may or may not be desirable. Could you explain a bit more about the implementation?

Reply:
I wanted them separate to minimize the impact of the  checks. These are just meant to be simple pod service checks internal to our Kube cluster to test whether routing should be done to the pod. 
These aren't meant for testing full application health which will be done through external pings for full page rendering.

Reply:
Is it possible to not have these exposed through the site's URLs? right now I can navigate to the `/status/db` in a browser and it shows the status response.

Reply:
Hi @jobara,

It won't work in your dev testing because the limits are in the webserver configuration which is different from the development environment you are using.

See `.deploy/accessibility-app/nginx.conf`  
```nginx
        # Status endpoints in Laravel
        location /status {
            allow 10.0.0.0/8; # Internal K8 network
            deny all;
            try_files $uri $uri/ /index.php?$query_string;
        }

        # Status endpoint for NGINX
        location /status/nginx {
            allow 10.0.0.0/8; # Internal K8 network
            deny all;
            stub_status on;
        }
```